app/models/kudo.rb

```diff
@@ -1,6 +1,7 @@
 class Kudo < ApplicationRecord
   belongs_to :giver, class_name: 'Employee'
   belongs_to :receiver, class_name: 'Employee'
+  belongs_to :company_value, class_name: 'CompanyValue'
```

Comment:
When the name of association matches the class you can simply write as:
```suggestion
  belongs_to :company_value
```